backend/api/Areas/Tools/Models/Import/BuildingEvaluationModel.cs

```diff
@@ -3,7 +3,7 @@ namespace Pims.Api.Areas.Tools.Models.Import
     public class BuildingEvaluationModel : Pims.Api.Models.BaseModel
     {
         #region Properties
-        public int PropertyId { get; set; }
+        public int BuildingId { get; set; }
```

Comment:
Note I made these match the database column names.  I don't know if this will breach @devinleighsmith your Property Detail View Page.  Additionally most of this is going to be updated/changed this sprint because we are changing how we implemented evaluations.

Reply:
It probably will. I'm fine with taking this and making the required adjustments on my side.